examples/mptoken/rpc/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/Peersyst/xrpl-go/pkg/crypto"
+	"github.com/Peersyst/xrpl-go/xrpl/faucet"
+	"github.com/Peersyst/xrpl-go/xrpl/rpc"
+	transactions "github.com/Peersyst/xrpl-go/xrpl/transaction"
+	"github.com/Peersyst/xrpl-go/xrpl/transaction/types"
+	"github.com/Peersyst/xrpl-go/xrpl/wallet"
+)
+
+func main() {
+	// Configure the client
+	cfg, err := rpc.NewClientConfig(
+		"https://s.altnet.rippletest.net:51234",
+		rpc.WithFaucetProvider(faucet.NewTestnetFaucetProvider()),
+	)
+	if err != nil {
+		panic(err)
+	}
+	client := rpc.NewClient(cfg)
+
+	fmt.Println("⏳ Funding wallets...")
+	// Create and fund the cold wallet (issuer)
+	issuerWallet, err := wallet.New(crypto.ED25519())
+	if err != nil {
+		fmt.Println("❌ Error creating cold wallet:", err)
+		return
+	}
+	if err := client.FundWallet(&issuerWallet); err != nil {
+		fmt.Println("❌ Error funding cold wallet:", err)
+		return
+	}
+	fmt.Println("💸 Cold wallet funded!")
+
+	// Create and fund the hot wallet (holder)
+	hotWallet, err := wallet.New(crypto.ED25519())
```

Comment:
We are not using `hotWallet` later so we can remove it.
Otherwise we should showcase the `MPTokenAuthorize` (using the `hotWallet`) and then a Payment using the MPT for a "full" cycle.

Reply:
That's what I have in mind, but I wanted to merge first the integration runner to test it in a local ledger